app/components/osf-navbar/home-links/component.ts

```diff
@@ -18,4 +19,8 @@ export default class OsfNavbarHomeLinks extends Component.extend({
     donateUrl = 'https://cos.io/donate';
     myProjectsUrl = serviceLinks.myProjects;
     reviewsUrl = serviceLinks.reviewsHome;
+    @action
+    closeSecondaryNav() {
```

Comment:
Instead of redefining this here, could you pass the action from `osf-navbar` in?
```hbs
{{! osf-navbar/template.hbs }}
{{component linksComponent onLinkClick=closeSecondaryNavigation}}

{{! osf-navbar/home-links/template.hbs }}
<li {{action onLinkClick}}> ... </li>

Reply:
Mmmmmaybe. I'll give that a try.

Reply:
💯